evo/tools/plot.py

```diff
@@ -346,7 +346,7 @@ def colored_line_collection(
 
 def traj_colormap(ax: plt.Axes, traj: trajectory.PosePath3D,
                   array: ListOrArray, plot_mode: PlotMode, min_map: float,
-                  max_map: float, title: str = "") -> None:
+                  max_map: float, title: str = "", fig: mpl.figure.Figure = None) -> None:
```

Comment:
`typing.Optional[pl.figure.Figure]`

Reply:
Alright that should hopefullly fix it. Thanks